saltlint/rules/HyphenHasSpaceRule.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Dawid Malinowski <dawidmalina@gmail.com>
+
+from saltlint.linter import SaltLintRule
```

Comment:
Pylint seems to fail on the order of importing. Please move `import re` above `from saltlint.linter import SaltLintRule`